packages/ozone/src/mod-service/index.ts

```diff
@@ -312,19 +313,6 @@ export class ModerationService {
       .executeTakeFirst()
   }
 
-  async getCurrentStatus(
```

Comment:
This work might just be in-progress, but I think this method is still needed. Specifically, I see it used in `createReport`

Reply:
This is a mistake. will revert 👍